WordPress/WordPressTest/NotificationSyncMediatorTests.swift

```diff
@@ -56,9 +56,6 @@ class NotificationSyncMediatorTests: CoreDataTestCase {
         // Make sure the collection is empty, to begin with
         XCTAssert(mainContext.countObjects(ofType: Notification.self) == 0)
 
-        // CoreData Expectations
-        let contextSaved = expectation(forNotification: .NSManagedObjectContextDidSave, object: mainContext)
```

Comment:
This expectation relies on an implementation detail of `ContextManager`, or `LegacyContextFactory` I should say. I deleted it here, since the rest of the assertions should be good enough to cover the unit test.